tests/cases/compiler/errorSpanForUnclosedJsxTag.tsx

```diff
@@ -0,0 +1,12 @@
+// @jsx: react
+import React from "react";
```

Comment:
You can replace this with `declare const React: any` to avoid import errors, especially since this is a syntactic test.

Reply:
Thanks a lot.